resources/views/courses/show.blade.php

```diff
@@ -0,0 +1,81 @@
+<x-app-layout page-width="wide">
+    <x-slot name="title">{{ $course->title }}</x-slot>
+    <x-slot name="header">
+        @if ($receivedCertificate)
+            <div class="notification flex justify-between">
+                <div class="my-auto">
+                    {{ __('You now have completed this course.') }}
+                </div>
+                @livewire('email-results', ['quiz' => $course->quiz])
+            </div>
+        @endif
+        <h1>
+            {{ $course->title }}
+        </h1>
+    </x-slot>
+    <div class="stack ml-2 mr-2">
+        <a href="{{ localized_route('resource-collections.index') }}">{{ __('Back') }}</a>
+        <x-slot name="title">{{ $course->title }}</x-slot>
+        <div class="flex justify-between">
+            <div>
+                {{ __('Author: :author', ['author' => $course->author ?? $course->organization?->name]) }}
+            </div>
+            <div>
+                {{ __('Resource type: Training') }}
+            </div>
+            <div>
+                {{ __('Published on: :created_at', ['created_at' => $course->created_at->isoFormat('MM, DD, YYYY')]) }}
```

Comment:
The date is coming up as 02, 23, 2023 instead of February 23, 2023. I think you mean to use the following:

```suggestion
                {{ __('Published on: :created_at', ['created_at' => $course->created_at->isoFormat('LL')]) }}
```